MonoGame.Framework/Audio/Xact/Cue.cs

```diff
@@ -305,6 +307,8 @@ internal void Update(float dt)
                     }
                 }
 
+                pitch = MathHelper.Clamp(pitch, -1.0f, 1.0f);
```

Comment:
This is wrong...  XACT allows pitches outside this range.  The correct clamping is done at the platform SoundEffectInstance implementation.